index.html

```diff
@@ -1477,9 +1476,9 @@ <h2>Public Keys</h2>
 
       <p>
 If a public key does not exist in the <a>DID document</a>, it MUST be assumed
-the key was revoked or is invalid. The <a>DID document</a> MAY contain revoked
-keys. A <a>DID document</a> containing a revoked key MUST also contain or refer
-to the revocation information for the key (for example, a revocation list). Each
+the key was revoked or is invalid.
+The <a>DID document</a> MAY NOT contain revoked keys.
```

Comment:
lol... this is exactly the PR I made a while ago. Hopefully this one succeeds. paging @jandrieu .

Reply:
I think we need to use "MUST" here. Furthermore, we may get approval from @jandrieu if we indicate that revoked keys must not appear via a verification relationship vs. must not appear at all in any place.

```suggestion
The <a>DID document</a> MUST NOT express revoked keys using a <a>verification relationship</a>.
```

Reply:
@selfissued ^